registration-processor/post-processor/registration-processor-printing-stage/src/main/java/io/mosip/registration/processor/print/stage/PrintingStage.java

```diff
@@ -374,8 +375,8 @@ public void start() {
 	public String generatePin() {
 		if (sr == null)
 			instantiate();
-		int randomInteger = sr.nextInt(max - min) + min;
-		return String.valueOf(randomInteger);
+		int secureRandomInteger = sr.nextInt(upperBound - lowerBound) + lowerBound;
```

Comment:
No this will create a huge bottleneck in the system. The keymanager has a pin generation why are we not using that api?

Reply:
As we have no api to generate a pin in keymanager we can use this as of now.

Reply:
Can we get it from our OTP api?

Reply:
or otp jar to generate a secure pin?